bin/alluxio

```diff
@@ -8,9 +8,12 @@ fi
 BIN=$(cd "$( dirname "$0" )"; pwd)
 
 function printUsage {
-  echo "Usage: alluxio COMMAND "
-  echo "where COMMAND is one of:"
-  echo -e "  format [-s]   \t Format Alluxio (if -s specified, only format if underfs is local and doesn't already exist)"
+  echo "Usage: alluxio COMMAND [GENERIC_OPTIONS] [COMMAND_OPTIONS]"
```

Comment:
as for @aaudiber's comment ... I think the following pattern is common:

`<command> <command_options> <subcommand> <subcommand_options> ...`

so to be consistent with that, the invocation we support should look like:

`alluxio [ALLUXIO_OPTIONS] COMMAND [COMMAND_OPTIONS]`

having said that, since not all the commands actually use -D (e.g. bootstrap-conf or killAll or copyDir), I would not call these out specifically ... I think you do that because of the limitations of parsing options in shell, correct?

this would be much easier in Go =_=